AlfrescoApp/Model/Managers/RealmSyncManager+Refresh.m

```diff
@@ -82,10 +84,10 @@ - (void)invalidateParentChildHierarchy
     [realm commitWriteTransaction];
 }
 
-- (void)rebuildDataBaseWithCompletionBlock:(void (^)(BOOL completed))completionBlock
+- (void)rebuildDataBaseWithArrayOfFilesToDownload:(NSMutableArray *)arrayToDownload arrayOfFilesToUpload:(NSMutableArray *)arrayToUpload withCompletionBlock:(void (^)(BOOL completed))completionBlock
 {
     RLMRealm *realm = self.mainThreadRealm;
-    RLMResults *topLevelFolders = [[RealmManager sharedManager] topLevelFoldersInRealm:realm];
+    RLMResults *topLevelFolders = [[RealmManager sharedManager] topLevelSyncNodesInRealm:realm];
```

Comment:
Confusing variable name; it doesn't match with the type of nodes returned by the topLevelSyncNodesInRealm method.